app/forms.py

```diff
@@ -35,43 +40,43 @@ def _safe(form, field):
 class RenderableForm(Form):
 
     def render_to_html(self):
-        return render_template("forms/generic.html", fields=self._fields.iteritems())
+        return render_template('forms/generic.html', fields=self._fields.iteritems())
 
 
 # WTForm objects that are used in rendering. Each Field in this object corresponds to the user-input columns in the DB
 class RegularPostForm(RenderableForm):
-    id = HiddenField("", [validators.DataRequired(), validators.Length(max=8)])
-    submitters_username = HiddenField("", [validators.DataRequired(), validators.Length(max=50)])
+    id = HiddenField('', [validators.DataRequired(), validators.Length(max=8)])
```

Comment:
Can you use a different name that doesn't collide with the built in keyword id? PEP8 would say to use `id_` or a different name altogether.